libs/colbert/ragstack_colbert/base_embedding.py

```diff
@@ -10,7 +10,7 @@
 from .chunks import EmbeddedChunk
 
 
-class TokenEmbeddings(ABC):
+class BaseEmbedding(ABC):
```

Comment:
So the class name should not be plural right?

Reply:
i think so... i keep going back and forth on this